docs/changes.rst

```diff
@@ -50,62 +50,115 @@ The best check that everything is ok is to run a dry-run extraction::
+  - --verify-data to verify data cryptographically on the client, #975
+- borg list, #751, #1179
 
   - removed {formatkeys}, see "borg list --help"
   - --list-format is deprecated, use --format instead
+  - --format now also applies to listing archives, not only archive contents, #1179
   - now supports the usual [PATH [PATHS…]] syntax and excludes
   - new keys: csize, num_chunks, unique_chunks, NUL
   - supports guaranteed_available hashlib hashes
-    (to avoid varying functionality depending on environment)
-- prune:
+    (to avoid varying functionality depending on environment),
+    which includes the SHA1 and SHA2 family as well as MD5
+- borg prune:
 
   - to better visualize the "thinning out", we now list all archives in
     reverse time order. rephrase and reorder help text.
   - implement --keep-last N via --keep-secondly N, also --keep-minutely.
     assuming that there is not more than 1 backup archive made in 1s,
     --keep-last N and --keep-secondly N are equivalent, #537
-- borg comment: add archive comments, #842
+  - cleanup checkpoints except the latest, #1008
+- borg extract:
+
+  - added --progress, #1449
+  - Linux: limited support for BSD flags, #1050
+- borg info:
+
+  - output is now more similar to borg create --stats, #977
+- repository:
+
+  - added progress information to commit/compaction phase (often takes some time when deleting/pruning), #1519
+  - automatic recovery for some forms of repository inconsistency, #858
+  - check free space before going forward with a commit, #1336
+  - improved write performance (esp. for rotating media), #985
+
+    - new IO code for Linux
+    - raised default segment size to approx 512 MiB
+  - improved compaction performance, #1041
+  - reduced client CPU load and improved performance for remote repositories, #940
+
+- options that imply output (--show-rc, --show-version, --list, --stats,
+  --progress) don't need -v/--info to have that output displayed, #865
+- add archive comments (via borg (re)create --comment), #842
 - provide "borgfs" wrapper for borg mount, enables usage via fstab, #743
-- create: add 'x' status for excluded paths, #814
-- --show-version: shows/logs the borg version, #725
 - borg list/prune/delete: also output archive id, #731
+- --show-version: shows/logs the borg version, #725
+- added --debug-topic for granular debug logging, #1447
+- use atomic file writing/updating for configuration and key files, #1377
+- BORG_KEY_FILE environment variable, #1001
+- self-testing module, #970
+
 
 Bug fixes:
 
+- list: fixed default output being produced if --format is given with empty parameter, #1489
+- create: fixed overflowing progress line with CJK and similar characters, #1051
+- prune: fixed crash if --prefix resulted in no matches, #1029
 - init: clean up partial repo if passphrase input is aborted, #850
 - info: quote cmdline arguments that have spaces in them
 - failing hashindex tests on netbsd, #804
```

Comment:
Looked at it again and am now more confused:

- f63be63 fixed byte order detection on illumos, and broke it on netbsd
- e494d2 fixed netbsd again
- both are between 1.0.0 and 1.0.1
- so this shouldn't be a changelog entry at all

```
$ git tag --contains f63be63
1.0.1
1.0.2
1.0.3
1.0.4
1.0.5
1.0.6
1.0.6rc1
1.0.7
1.0.7bin
1.0.7rc1
1.0.7rc2
1.1.dev0

git tag --contains e494d2 
1.0.1
1.0.2
1.0.3
1.0.4
1.0.5
1.0.6
1.0.6rc1
1.0.7
1.0.7bin
1.0.7rc1
1.0.7rc2
1.1.dev0
```